overrides/home.html

```diff
@@ -283,6 +283,20 @@ <h1>Case Studies</h1>
             </p>
           </a>
         </div>
+        <div class="col">
+          <a href="{{ 'about/case-studies/pnc/' | url }}">
+            <div class="logo">
+              <img class="pnc" src="images/case-studies/pnc_bank.png" alt="" draggable="false" />
+            </div>
+            <hr>
+            <p class="normal-text">
+              “Our real success was our ability to say if your code change is fully compliant and does not affect complex authentication or funds transfer, you can go from new code to production in just the time it takes to run your pipelines. There is no longer 120 hours over 37 days of non-code compliance work standing in the way of production. Thousands of PNC software developers have seen their deployment window shrink to near real time. Our teams achieve true continuous delivery. That is our big win.”<br><br>
```

Comment:
This doesn't render well on the page, with the outer edges on both sides getting cut off a bit: 

![image](https://user-images.githubusercontent.com/44698588/226146720-a0e734b4-e07b-48e6-a5c1-0e2df076bf41.png)

Saw this on both Chrome and Firefox... I wonder if there's a way to pad some space on the edges? (@gabo1208 , am I remembering right that you had helped on this before?)

Reply:
In the `home.css` file, if you change

```
.case-studies-container .col {
  min-width: 38%;
  max-width: 40%;
}
```

to 
```
.case-studies-container .col {
  min-width: 30%;
  max-width: 40%;
}
```

that seems to make it render better. However that's just a quick fix, the homepage honestly needs to be designed better so that it's scalable and responsive.

Reply:
Yeah I used `max-width: 120pt;` in some of the home sections to do that. Sorry the delay